storio-common-annotations-processor/build.gradle

```diff
@@ -1,18 +1,22 @@
-apply plugin: 'java'
-
-targetCompatibility = '1.6'
-sourceCompatibility = '1.6'
+apply plugin: 'kotlin'
 
 dependencies {
     compile libraries.intellijAnnotations
     compile libraries.autoService
     compile libraries.javaPoet
+    compile libraries.kotlinStdLib
```

Comment:
I don't remember the exact number of methods in Kotlin stdlib, but it doesn't matter anyway because this dependency isn't going to be in apk so dex limit won't be affected.